system-int-test/test/cmr/system_int_test/search/collection_cloud_hosted_search_test.clj

```diff
@@ -0,0 +1,49 @@
+(ns cmr.system-int-test.search.collection-cloud-hosted-search-test
+  "Integration tests for searching for records that are cloud hosted"
+  (:require
+   [clojure.test :refer :all]
+   [cmr.mock-echo.client.echo-util :as echo]
+   [cmr.indexer.data.concepts.tag :as itag]
+   [cmr.system-int-test.data2.core :as data2]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.system :as system]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]
+   [cmr.system-int-test.utils.search-util :as search]
+   [cmr.system-int-test.utils.tag-util :as tags]))
+
+(use-fixtures :each (join-fixtures
+                      [(ingest/reset-fixture {"provguid1" "PROV1"})
+                       tags/grant-all-tag-fixture]))
+
+(defn- create-direct-dist-info
+  "Create a Direct Distribution"
+  []
+  {:DirectDistributionInformation {:Region "us-east-1"
+                                   :S3BucketAndObjectPrefixNames ["Bucket"]
+                                   :S3CredentialsAPIEndpoint "https://example.org"
+                                   :S3CredentialsAPIDocumentationURL "https://example.org"}})
+
+(deftest search-collections-that-are-cloud-hosted
+  (let [coll1 (data2/ingest-umm-spec-collection "PROV1" (data-umm-c/collection 1 {}))
+        coll2 (data2/ingest-umm-spec-collection "PROV1" (data-umm-c/collection 2 {}))
+        coll3 (data2/ingest-umm-spec-collection "PROV1" (data-umm-c/collection 3 (create-direct-dist-info)))]
+    (index/wait-until-indexed)
+
+    (testing "Search by the cloud_hosted flag"
+      (are [items value]
```

Comment:
don't use are, it was deprecated long ago.  Change the tests to use are3.  

Here is an example from cmr.umm-spec.test.xml-to-umm-mappings.iso-shared.distributions-related-url
(deftest description-string-parsing-keyword-test
  (testing (str "Parsing given string and converting it to a map then converting the left hand"
                "strings to keywords.")
    (are3 [string expected]
      (is (= expected (sru/parse-url-types-from-description string)))

      "ISO MENDS Collection Description string"
      "URLContentType: DistributionURL Description: NASA's newest search and order tool for subsetting, reprojecting, and reformatting data. Type: GET DATA Subtype: Earthdata Search"
      {:Type "GET DATA",
       :URLContentType "DistributionURL",
       :Description "NASA's newest search and order tool for subsetting, reprojecting, and reformatting data.",
       :Subtype "Earthdata Search"}

      "String with odd and nil values"
      ":: URLContentType:nil Checksum: \"nil\" Description: NASA's newest lawnmower ascii art: __\\.-.,,,, Type: SELF PROPELLED Subtype: Earthdata Search"
      {:Checksum "\"nil\"",
       :Description "NASA's newest lawnmower ascii art: __\\.-.,,,,",
       :Type "SELF PROPELLED",
       :Subtype "Earthdata Search"}

      "String with No pattern match"
      "This is just a plain description."
      {:Description "This is just a plain description."})))